src/sagemaker/jumpstart/artifacts.py

```diff
@@ -42,13 +42,14 @@ def _retrieve_image_uri(
 ):
     """Retrieves the container image URI for JumpStart models.
 
-    Only `model_id` and `model_version` are required to be non-None;
+    Only `model_id`, `model_version`, and `image_scope` are required;
```

Comment:
why not removing the `Optional` typing for `image_scope` and move it as the 3rd positional argument then?